torch/onnx/_internal/fx/passes/modularization.py

```diff
@@ -818,6 +835,17 @@ class Modularize(_pass.Transform):
 
     """
 
+    @_beartype.beartype
+    def __init__(
+        self,
+        diagnostic_context: diagnostics.DiagnosticContext,
+        module: torch.fx.GraphModule,
+        is_exported_program: bool = False,
+    ):
+        super().__init__(diagnostic_context, module)
+        self.module = module
+        self.is_exported_program = is_exported_program
```

Comment:
I wonder if there is any attribute or pattern that we could use to detect that `module` comes from torch.export.export without the user manually saying it

Reply:
Tried multiple ways to find the difference between their graph modules. It looks like the `nn_module_stack` from `ExportedProgram` would be one more than nn.Module. But there is no guarantee that the nodes from nn.Module gm would have nodes that only contains one layer of nn_module_stack. It's risky to check whether it's ExportedProgram or not that way.. So I incline to keep this for now.